src/components/common/Tabs/EditProfileTab/index.js

```diff
@@ -0,0 +1,98 @@
+import Back from 'components/common/Back';
+import Button from 'components/common/Button';
+import useTranslation from 'hooks/useTranslation';
+import { useForm } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+import { error, success } from 'react-toastify-redux';
+import { selectAuth, useChangePasswordMutation, useUpdateUserMutation } from 'services/auth/auth';
+import { setHomeTab } from 'state/slices/tabSlice';
+import { tabsEnum } from '..';
```

Comment:
Maybe it's something I'm not familiared with but what is this import path?

Reply:
It's a relative route, such as "../folder/file", it references the parent folder